src/pages/Books.js

```diff
@@ -1,16 +1,14 @@
-import Button from 'react-bootstrap/Button';
 import Book from '../Components/Book';
+import Booksform from '../Components/booksform';
 
```

Comment:
- [ ] As suggested by the previous reviewer, It will be better if your books component uses the book component to render a list of book by passing props i.e data (title, author) into your pages folder Books.js component.

## EXAMPLE

```
class Books extends React.PureComponent {
  render() {
    return (
      <div>
        <Book title="title1" author="author1" />
        <Book title="title2" author="author2" />
        <AddNewBookForm />
      </div>
    );
  }
```

Reply:
ok now im using props types to put that info on the screen

Reply:
@cvalencia1991, I kindly suggest that you submit again for rereview.